doc/user-guide/github.md

```diff
@@ -69,4 +76,28 @@ Without `--watch`, the logs will look like this:
 
 ![](../static/github/logs.png)
 
-A complete sample project is [available here.](https://github.com/edublancas/cloud-template)
\ No newline at end of file
+A complete sample project is [available here.](https://github.com/edublancas/cloud-template)
+
+## Configure action through CLI
+
+If your project is already hosted on Github, running `ploomber-cloud init` inside your repository folder will initialize
+the project and prompt for configuring Github action:
+
+```bash
+Your app 'snowy-disk-4284' has been configured successfully!
+Do you want to configure a github action? [y/N]: 
+```
+
+On confirming with `y` the CLI will create a `ploomber-cloud.yaml` file in the path `.github/workflows`.
+In order to trigger an action for deploying the project using Github actions you need to add, commit and push this file along with the `ploomber-cloud.json`.
+
+Once done, you can monitor progress as discussed [above](monitor). Ensure that the API key is set as Github secret.
+
+In case the [workflow template](https://github.com/edublancas/cloud-template/blob/main/.github/workflows/ploomber-cloud.yaml) has been updated, and you need to re-initialise the application the CLI will prompt your for updating the workflow file:
```

Comment:
this point is unclear: project re-initialization doesn't happen often. the only case I can think of is if the project gets corrupted.

wouldn't it make more sense to check for an updated workflow file when calling `ploomber-cloud deploy`?

I see two scenarios:

1. if `ploomber-cloud deploy` is running on github actions, then we can juts print a mesage
2. if `ploomber-cloud deploy` is running locally, we can deploy and then prompt the user, asking if they want to update their ploomber-cloud.yaml

Reply:
Yes checking during deploy would make more sense. For scenario 1, the message will be displayed in the github actions logs?

Reply:
I thought through this, here's my feedback (note that this workflow is different to what I proposed earlier cause I noticed some potential issues):

- we can add a `ploomber-cloud github` command that can take care of configuring for the first time (if the file doesn't exist) or update the workflow (if it already exists)
- then, when users run `ploomber-cloud init` or `ploomber-cloud deploy`, we can either thell them they can configure github deployment (or update their existing workflow) by running `ploomber-cloud github`

the reason I'm proposing this is because I feel like prompting users for an answer when running `init` or `deploy` might be annoying and unexpected. instead, we can just print some relevant message on `init`/`deploy` and they can decide if they want to configure/update github actions by running `ploomber-cloud github`

Last call is up to you; think what scenarios users will encounter and define the workflow that makes more sense

Reply:
Sure! I'll start implementing this and if I find any scenario which would be an issue , I'll discuss and modify accordingly.

Reply:
I think it's ok to tell the user to update only when the contents are different, no need for more complex implementation right now

Reply:
Please refer to this [comment](https://github.com/ploomber/cli/pull/22#issuecomment-1904217040)